.travis.yml

```diff
@@ -9,9 +11,18 @@ env:
   - ADMIN_PARTY=false
 
 services:
-  - couchdb
+  - docker
+
+before_install:
+  - docker pull apache/couchdb:2.1.1
+  - docker run -d -it -p 5984:5984 apache/couchdb:2.1.1 --with-admin-party-please --with-haproxy
 
 install: "pip install -r requirements.txt && pip install -r test-requirements.txt"
+
+before_script:
+    - curl -X PUT http://localhost:5984/_users
```

Comment:
You'll need a line in here to make sure that the endpoint is up before trying to create the DBs, otherwise we may well run into timing problems later.
Something like https://github.com/cloudant/sync-cordova-plugin/blob/master/.travis.yml#L66

Reply:
Added in 62ad10e.